mlflow/johnsnowlabs/__init__.py

```diff
@@ -348,9 +358,14 @@ def log_model(
             pip_requirements=pip_requirements,
             extra_pip_requirements=extra_pip_requirements,
             metadata=metadata,
+            params=params,
+            tags=tags,
+            model_type=model_type,
+            step=step,
+            model_id=model_id,
         )
     # Otherwise, override the default model log behavior and save model directly to artifact repo
-    mlflow_model = Model(artifact_path=artifact_path, run_id=run_id)
+    mlflow_model = Model(artifact_path=name, run_id=run_id)
```

Comment:
need to figure out how to make this work.

Reply:
We should make `artifact_path` optional in `Model`, if that helps...